config/boards/station-p1.csc

```diff
@@ -1,7 +1,7 @@
 # Rockchip RK3399 hexa core 4GB LPDDR4 SoC eMMC GBE USB3
 BOARD_NAME="Station P1"
-BOARDFAMILY="media"
-BOARD_MAINTAINER="150balbes"
+BOARDFAMILY="rockchip64"
+BOARD_MAINTAINER=""
```

Comment:
If you have changed the code, then you have taken on the burden of the maintainer and must provide your name.

Reply:
It is CSC so no dedicated maintainer needed.

Reply:
please stop asking for these changes. Igor has a magic thing uses some secret database of contributors. Changing it in PRs makes no difference, since there will always come [something like this](https://github.com/armbian/build/pull/6072) after a while.